config/default.yml

```
@@ -53,6 +53,13 @@ Sorbet/FalseSigil:
   - db/**/*.rb
   - script/**/*
 
+Sorbet/ForbidExtendTSigInShims:
+  Description: 'Forbid the use of `extend T::Sig` in RBI shims'
+  Enabled: false
+  VersionAdded: '0.86'
+  Include:
+  - "sorbet/rbi/shims/**/*.rbi"
```

I think it's safe to include all kind of RBI by default. Even generator should not do this.

```suggestion
  - "**/*.rbi"
```